test/wGovernanceOLAS.js

```diff
@@ -0,0 +1,424 @@
+/*global describe, context, beforeEach, it*/
+
+const { expect } = require("chai");
```

Comment:
New test files are basically clones of original ones, with the veOLAS wrapper being called instead of veOLAS where applicable